access-control-app/src/cmr/access_control/system.clj

```diff
@@ -84,7 +84,7 @@
                          `system-holder
                          [(af/refresh-acl-cache-job "access-control-acl-cache-refresh")
                           jvm-info/log-jvm-statistics-job])}]
-    (transmit-config/system-with-connections sys [:echo-rest :metadata-db :urs :cubby])))
+    (transmit-config/system-with-connections sys [:access-control :echo-rest :metadata-db :urs :cubby])))
```

Comment:
I agree, but there was a reason I had to do this which I cannot recall at the moment... removing and will see what breaks